.github/scripts/update_dbeaver_drivers.py

```diff
@@ -29,5 +29,9 @@
 output["postgis_geometry"] = postgis
 output["postgis_jdbc"] = postgis
 
+remote_page = requests.get("https://repo1.maven.org/maven2/com/github/waffle/waffle-jna/", allow_redirects=True)
+root = html.fromstring(remote_page.content)
+output["waffle_jna"] = natsorted([v for v in root.xpath("//a[contains(text(), '/')]/@href") if v != "../"])[-1].replace("/", "")
```

Comment:
It would be good to explain what is happening here. It isn't very clear to me what this is doing.

There is a lot of duplicate code in the script, so this process would be worth generalising in a function.

Reply:
Great, that is much tidier :rocket: